packages/math-wallet/src/lib/math-wallet.ts

```diff
@@ -155,6 +155,7 @@ export const setupMathWallet = ({
         downloadUrl:
           "https://chrome.google.com/webstore/detail/math-wallet/afbcbjpbpfadlkmhmclhkeeodmamcflc",
         deprecated: false,
+        available: installed,
```

Comment:
We'll need to modify the `setupInstance` method in `wallet-modules` to avoid calling `init` when a wallet is unavailable. We run the risk of causing crashes otherwise 🙁